pyfr/solvers/euler/kernels/entropyfilter.mako

```diff
@@ -17,6 +17,19 @@
         ${pyfr.expand('compute_entropy', 'ui', 'd', 'p', 'e')};
         dmin = fmin(dmin, d); pmin = fmin(pmin, p); emin = fmin(emin, e);
     }
+
+    % if not fpts_in_upts:
+    fpdtype_t uf[${nvars}];
+    for (int fidx = 0; fidx < ${nfpts}; fidx++)
+    {
+        % for vidx in range(nvars):
+        uf[${vidx}] = ${pyfr.dot('m0[fidx][{k}]', f'u[{{k}}][{vidx}]', k=nupts)};
```

Comment:
Depends an implicit definition of m0.  No great solutions here, though.

Reply:
Yeah, we do the same with the vdm matmul as well.

Reply:
vdm we can fix easily enough with an extern